src/routes/(app)/members/[studentId]/+page.svelte

```diff
@@ -123,4 +125,35 @@
       {/if}
     </div>
   </div>
+  <div
+    class="col-span-5 sm:col-span-3 sm:col-start-3 md:col-start-1 lg:col-span-2 lg:col-start-1 xl:col-span-1 xl:col-start-1"
+  >
+    {#if isMe}
```

Comment:
Would prefer `{#if isMe && data.allMemberDoors.size > 0}` so we don't render something empty